src/components/accessories/patientNewBill/PatientNewBill.tsx

```diff
@@ -0,0 +1,226 @@
+import { scrollToElement } from "../../../libraries/uiUtils/scrollToElement";
+import { useHistory } from "react-router";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Backdrop,
+  CircularProgress,
+  Theme,
+} from "@material-ui/core";
+import { useStyles } from "./consts";
+import { newBillReset, updateBillReset } from "../../../state/bills/actions";
+
+const PatientNewBill: FC = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+
+  const [openPayment, setOpenPayment] = useState(false);
+
+  const {
+    fullBill,
+    bill,
+    billItems,
+    billPayments,
+    itemsRowData,
+    billTotal,
+    paymentTotal,
+    itemToEdit,
+    creationMode,
+    status,
+    saveBill,
+    setItemToEdit,
+    handleBillEdit,
+    handleAddItem,
+    handleEditItem,
+    handleAddPayment,
+    handleDeleteItem,
+    handleDeletePayment,
+  } = useFullBill();
+
+  const {
+    showItemPicker,
+    showPaymentDialog,
+    openSaveDialog,
+    handleSaveDialog,
+    handleItemPicker,
+    handlePaymentDialog,
+  } = useDialogStatus();
+
+  const onSubmitItem = (item: BillItemsDTO, isNew: boolean) => {
+    isNew ? handleAddItem(item) : handleEditItem(item);
+    handleItemPicker();
+  };
+
+  const handleItemDialog = () => {
+    handleItemPicker();
+    if (itemToEdit) {
+      setItemToEdit(undefined);
+    }
+  };
+
+  const infoBoxRef = useRef<HTMLDivElement>(null);
+  const history = useHistory();
+  const { patient } = useSelectedPatient();
+
+  const resetItemFormCallback = () => {};
+
+  const handleTableEdit = useCallback((row) => {
+    setItemToEdit(row);
+    handleItemPicker();
+  }, []);
+
+  const handlePayment = (values: Record<string, any>) => {
+    handleAddPayment(values);
+    handlePaymentDialog();
+  };
+
+  useEffect(() => {
+    dispatch(getPrices());
+  }, [dispatch]);
+
+  useEffect(() => {
+    console.log(status);
+    if (status === "FAIL") {
+      scrollToElement(infoBoxRef.current);
+    }
+    if (status === "SUCCESS") {
+      history.push(`/details/${patient.code ?? ""}/billsrecord`);
+    }
+  }, [status]);
+
+  const classes = useStyles();
+
+  return (
+    <>
+      <div className="patientNewBill">
+        <div className="patientNewBill_main">
+          {billItems.length != 0 && (
+            <div className="patientNewBill_payment">
+              <Accordion expanded={openPayment}>
+                <AccordionSummary onClick={() => setOpenPayment(!openPayment)}>
+                  <FilterList fontSize="small" />
+                  <h5>{t("bill.payment")}</h5>
+                </AccordionSummary>
+                <AccordionDetails>
+                  <ItemPayment
+                    saveBill={saveBill}
+                    handlePaymentDialog={handlePaymentDialog}
+                    billTotal={billTotal}
+                    paymentTotal={paymentTotal}
+                  />
+                </AccordionDetails>
+              </Accordion>
```

Comment:
Now it looks like fine.

Some changes:

1. label : "Payment recap"
2. it shoul'd be an accordion. The UI should be the same as now but without the logic open/close because this should be always expanded, I think the <Card> component is what we are looking for.

Reply:
Hi Alex ! It's done here. Merged conflicts have also been resolved. Please check it out.